catalog-rest-service/src/main/java/org/openmetadata/catalog/Entity.java

```diff
@@ -107,7 +107,19 @@
   //
   // List of entities whose changes should not be published to the Activity Feed
   //
-  public static final List<String> ACTIVITY_FEED_EXCLUDED_ENTITIES = List.of(USER, TEAM, ROLE, POLICY, BOTS);
+  public static final List<String> ACTIVITY_FEED_EXCLUDED_ENTITIES =
+      List.of(
+          USER,
+          TEAM,
+          ROLE,
+          POLICY,
+          BOTS,
```

Comment:
BOTs should be there @vivekratnavel

Reply:
@harshach Yes, this is bit entity and not not the bot users